tools/docker/tests/test_flows.sh

```diff
@@ -59,7 +59,7 @@ test_initial_ap_config() {
     MAC_AGENT1=$(echo $mac_agent1 | tr a-z A-Z)
     # Configure the controller and send renew
     send_CAPI_command gateway "DEV_RESET_DEFAULT" $redirect
-    send_CAPI_command gateway "DEV_SET_CONFIG,bss_info1,$MAC_AGENT1 8x Multi-AP-24G-1 0x0020 0x0008 maprocks1 0 1,bss_info2,$mac_agent1 8x Multi-AP-24G-2 0x0020 0x0008 maprocks2 1 0" $redirect
+    send_CAPI_command gateway "DEV_SET_CONFIG,bss_info1,$MAC_AGENT1 8x Multi-AP-24G-1 0x0020 0x0008 maprocks1 0 1,bss_info2,$MAC_AGENT1 8x Multi-AP-24G-2 0x0020 0x0008 maprocks2 1 0" $redirect
```

Comment:
NACK this. The whole point of this MAC_AGENT1 is to verify that it is really case insensitive, so we want *both* upper and lower case version in the same configuration.